c2_utils/src/mfx_defaults.cpp

```diff
@@ -197,6 +200,17 @@ mfxStatus mfx_set_defaults_mfxVideoParam_enc(mfxVideoParam* params)
             params->mfx.NumSlice = 1;
             params->mfx.NumRefFrame = 1;
             break;
+        case MFX_CODEC_VP9:
+            params->mfx.CodecProfile = MFX_PROFILE_VP9_0;
+            params->mfx.CodecLevel = MFX_LEVEL_UNKNOWN;
+            params->mfx.TargetUsage = MFX_TARGETUSAGE_BEST_SPEED;
+            params->mfx.FrameInfo.PicStruct = MFX_PICSTRUCT_PROGRESSIVE;
+            res = mfx_set_RateControlMethod(MFX_RATECONTROL_CBR, params);
+            params->mfx.GopPicSize = 16;
+            params->mfx.GopRefDist = 1;
+            params->mfx.NumSlice = 1;
+            params->mfx.NumRefFrame = 1;
```

Comment:
Could you please add below line here? VP9 HW encoder is using low power mode with VDEnc + Pak + HuC/GuC.
params->mfx.LowPower = MFX_CODINGOPTION_ON;